src/Apps/Artwork/Components/ArtworkDetails/index.tsx

```diff
@@ -36,3 +39,45 @@ export const ArtworkDetailsFragmentContainer = createFragmentContainer(
     }
   `
 )
+
+export const ArtworkDetailsQueryRenderer = ({
+  artwork,
+}: {
+  artwork: string
+}) => {
+  return (
+    <ContextConsumer>
+      {({ user, mediator, relayEnvironment }) => {
+        if (!user) {
+          return null
+        }
```

Comment:
Is a user necessary here?

Reply:
Oh... no, actually not at all. All this info should be available for not logged in people. Will update